src/code/actions.js

```diff
@@ -295,25 +295,23 @@ function _rejectEggFromBasket(eggIndex, basketIndex) {
   };
 }
 
-export function rejectEggFromBasket() {
-  return (dispatch, getState) => {
-    const { submittedEggIndex, submittedBasketIndex } = getState();
-    dispatch(_rejectEggFromBasket(submittedEggIndex, submittedBasketIndex));
+export function rejectEggFromBasket(args) {
+  return (dispatch) => {
+    dispatch(_rejectEggFromBasket(args.eggDrakeIndex, args.basketIndex));
   };
```

Comment:
Small detail: I think there's no need to use redux-thunk here to call a function with dispatch. That's normally only used if (1) you need to dispatch multiple actions, or (2) you need access to the `getState` argument.

Here it looks like you could just do

    export function rejectEggFromBasket(args) {
       return {
         type: actionTypes.EGG_REJECTED,
         eggIndex: args.eggIndex,
         basketIndex: args.basketIndex
       };
    }

and ditto for `_acceptEggInBasket`. (It looks like maybe this was just left over from when you needed access to `getState` earlier.)